scripts/data_migration/migrate_assessment_subject_day_data.js

```diff
@@ -0,0 +1,80 @@
+const { MongoClient } = require('mongodb')
+
+const COLLECTION_NAME = 'assessmentSubjectDayData'
+
+export async function migrateAssessmentSubjectDayData() {  
+  const mongoConnection = await MongoClient.connect(process.env.MONGODB_URI, {
+    ssl: process.env.NODE_ENV === 'production',
```

Comment:
is this something that will happen..?

Reply:
I think so? It certainly uses SSL in the current system but I'm not sure how things will work in AWS since Mongo will be on a private subnet